examples/performance-example/app/src/main/java/com/example/bugsnag/performance/MainActivity.kt

```diff
@@ -16,20 +29,37 @@ import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.drawWithContent
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
 import com.bugsnag.android.performance.BugsnagPerformance
 import com.bugsnag.android.performance.SpanOptions
-import com.bugsnag.android.performance.compose.MeasuredComposable
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.launch
 
 class MainActivity : AppCompatActivity() {
     private val network = ExampleNetworkCalls(this)
 
+    @OptIn(ExperimentalMaterial3Api::class)
     override fun onCreate(savedInstanceState: Bundle?) {
+        enableEdgeToEdge()
         super.onCreate(savedInstanceState)
         setContent {
-            MeasuredComposable(name = "MainActivity") {
-                AllButtons(network = network)
+            Scaffold(
```

Comment:
We still want the `Scaffold` to be measured, so it should be wrapped in the `MeasuredComposable`